src/main/webapp/app/exam/manage/student-exams/student-exams.component.ts

```diff
@@ -192,23 +192,6 @@ export class StudentExamsComponent implements OnInit, OnDestroy {
         });
     }
 
-    /**
-     * Starts all the exercises of the student exams that belong to the exam
-     */
-    startExercises() {
-        this.isLoading = true;
-        this.examManagementService.startExercises(this.courseId, this.examId).subscribe({
-            next: () => {
-                this.alertService.success('artemisApp.studentExams.startExerciseSuccess');
```

Comment:
Same as for the button's caption: The i18n key `artemisApp.studentExams.startExerciseSuccess` was only used here. Therefore, this key is not in usage anymore, is it? If so, it should be removed from the `exam.json` for both languages.

Reply:
done